simcase-game/src/views/Ranking.js

```diff
@@ -2,27 +2,182 @@
 import React from "react";
 import { jsx } from "@emotion/core";
 import { Link } from "@reach/router";
+import Header from "../components/Header";
 
-function Ranking() {
-  const buttonOptions = {
-    padding: 20,
-    textAlign: "center",
-    background: "rgb(74,0,255)",
-    color: "#fff",
-    textDecoration: "none",
-    margin: 10
-  };
+const leaderboardDummyData = {
+  1: { id: 1, username: "ccachay", points: 961 },
+  2: { id: 2, username: "dconde", points: 871 },
+  3: { id: 3, username: "jwayne", points: 841 },
+  4: { id: 4, username: "wsmith", points: 771 },
+  5: { id: 5, username: "sstallone", points: 762 },
+  6: { id: 6, username: "dcuevas", points: 714 },
+  7: { id: 7, username: "jperez", points: 601 }
+};
 
+const contentStyle = {
+  height: "75vh",
+  overflow: "auto",
+  color: "#4d4d4d",
+  textAlign: "center"
+};
+
+const tableStyle = {
+  width: "85%",
+  margin: "0 auto",
+  borderCollapse: "collapse",
+  fontSize: 16
+};
+
+const captionStyle = {
+  fontWeight: "bold",
+  fontSize: 18,
+  padding: 5
+};
+
+const col_1Style = {
+  width: "20%"
+};
+
+const col_2Style = {
+  // width: "50%"
```

Comment:
Be careful, don't forget to delete the commented line, and Isn't necessary to add `Style` name in the `const`, I think it's refactor styles